source/connect.txt

```diff
@@ -267,14 +267,11 @@ To disconnect from a deployment and exit ``mongosh``, you can:
 Limitations
 -----------
 
-- ``mongosh`` has the following 
-  :manual:`Kerberos authentication </core/kerberos/>` limitations:
-
-  - ``authMechanismProperties=CANONICALIZE_HOST_NAME:true|false`` is not
-    permitted in the connection string. Instead, specify hostname 
```

Comment:
This used to be true but isn’t anymore. Users should just do `?authMechanismProperties=CANONICALIZE_HOST_NAME:forward/forwardAndReverse/none`, never `gssapiCanonicalizeHostName`.

Reply:
Done